script2.js

```diff
@@ -0,0 +1,31 @@
+let emotions = {
+    upset: 0,
+    unhappy: 0,
+    neutral: 0,
+    happy: 0,
+    joyful: 0,
+    total: 0
+  };
+  
+  function submitSentiment(event) {
+    //get the emotion submitted
+    const selectedEmotion = document.querySelector('input[name="feeling"]:checked').value;
+  
+    //update the counts of the emotion and total
+    emotions[selectedEmotion]++;
+    emotions["total"]++;
+  
+    
+    
+    const breakdown = document.getElementById("breakdown");
+    Object.keys(emotions).forEach(emotion => {
+    if (emotion !== "total") {
+      const pctElement = document.getElementById(`${emotion}_pct`);
+      const percentage = emotions[emotion] / emotions["total"] * 100;
+      pctElement.textContent = `${percentage.toFixed(2)}% of people are feeling ${emotion}`;
```

Comment:
Small detail here, but when the page first loads, this won't be displayed until someone submits the form. One idea for a fix could be adding an eventListener for when the window loads that calculates the percentages (initially all 0) and updates the display